src/sxastarter/src/pages/api/robots.ts

```diff
@@ -4,7 +4,8 @@ import { siteResolver } from 'lib/site-resolver';
 import config from 'temp/config';
 
 const robotsApi = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
-  res.setHeader('Content-Type', 'text/plain');
+  // Ensure response is text/html
+  res.setHeader('Content-Type', 'text/html;charset=utf-8');
```

Comment:
You'll want to revert this change. Looks like this didn't make it into the JSS 21.2.4 patch release.